src/diffusers/models/unet_3d_condition.py

```diff
@@ -502,7 +502,6 @@ def forward(
         sample: torch.FloatTensor,
         timestep: Union[torch.Tensor, float, int],
         encoder_hidden_states: torch.Tensor,
-        class_labels: Optional[torch.Tensor] = None,
```

Comment:
Why do we delete this? This is backward breaking.

If `class_labels` is not needed, let's add a `kwarg`  that gracefully deprecates `class_labels`